src/algebra/continued_fractions/computation/approximations.lean

```diff
@@ -297,18 +303,299 @@ begin
   { have : g.partial_denominators.nth n = none, by rwa seq.terminated_at at terminated,
     have : g.terminated_at n, from
       terminated_at_iff_part_denom_none.elim_right (by rwa seq.terminated_at at terminated),
-    have : (gcf.of v).denominators (n + 1) = (gcf.of v).denominators n, from
+    have : g.denominators (n + 1) = g.denominators n, from
       denominators_stable_of_terminated n.le_succ this,
     rw this },
   { obtain ⟨b, nth_part_denom_eq⟩ : ∃ b, g.partial_denominators.nth n = some b, from
       option.ne_none_iff_exists'.mp not_terminated,
     have : 1 ≤ b, from of_one_le_nth_part_denom nth_part_denom_eq,
     calc
-      (gcf.of v).denominators n ≤ b * (gcf.of v).denominators n   : by simpa using
-                                                                      mul_le_mul_of_nonneg_right
-                                                                      this zero_le_of_denom
-                            ... ≤ (gcf.of v).denominators (n + 1) : le_of_succ_nth_denom
-                                                                      nth_part_denom_eq }
+      g.denominators n ≤ b * g.denominators n   : by simpa using (mul_le_mul_of_nonneg_right
```

Comment:
A common trick to stay within the line length limit is to format the first line as
```
calc some_long_left_hand_side
    ≤ some_right_hand_side_1 : _
... ≤ another_thingy_2 : _
```
you might consider using that here.